### PR TITLE
Add Cloud Extensibility Proxy (CEP) and vRealize Orchestrator Functions

### PR DESCRIPTION
**Pull request by GaryJBlake**
- Enhancement to Request-CSPToken
- Added Get-vROVersion with support for standalone, embedded and extensibility instances
- Added Get-CEPWorkflow to retrieve Workflows from CEP vRO Instance
- Added Invoke-CEPWorkflow to start a Workflow on CEP vRO Instance
- Added Get-CEPWorkflowExecution to get the Workflow Execution ID
- Added Get-CEPWorkflowExecutionState to get the Workflow Execution State
- Added Add-CEPTrustedCertificate to add a trusted cert to vRO

Signed-off-by: Gary Blake <gblake@vmware.com>
**Type of Pull Request**

- [ ] This is a bug fix.
- [x] This is an enhancement or feature.
- [ ] This is a code style / formatting update.
- [ ] This is a documentation update.
- [ ] This is a refactoring update.
- [ ] This is something else.
      Please describe:

**Related to Existing Issues**

Issue Number: N/A

**Test and Documentation Coverage**

- [x] Tests have been completed (for bug fixes / features).
- [ ] Documentation has been added / updated (for bug fixes / features).

**Breaking Changes?**

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### CODE REVIEW

File: PowerValidatedSolutions.psm1 [hunk 1]
```diff
@@ -29685,7 +29705,689 @@ Function Get-CSPPoxyAppliance {
...
+                }
+            ]
+        }
+        "@
+
+        Invoke-vROWorkflowOnCEP -id 3f23f186-158a-4869-b464-b7271fc216ba -parameters ($parameters | ConvertFrom-Json).parameters
+
+        .EXAMPLE
+        $param1 = New-vROParameterDefinition -name 'foo' -value 'bar' -type string -scope LOCAL
+        Invoke-vROWorkflowOnCEP -id 3f23f186-158a-4869-b464-b7271fc216ba -parameters $param1
```
Comment (tenthirtyam) [inline PowerValidatedSolutions.psm1:29959]: `Invoke-CEPWorkflow`

File: PowerValidatedSolutions.psm1 [hunk 2]
```diff
@@ -29685,7 +29705,689 @@ Function Get-CSPPoxyAppliance {
...
+
+        .EXAMPLE
+        Invoke-vROWorkflowOnCEP -name 'Import a trusted certificate from a file' | Invoke-vROWorkflowOnCEP -parameterName 'foo' -parameterValue 'bar' -parameterType string
```
[the remark this answers is not in the document]
Comment (GaryJBlake) [reply PowerValidatedSolutions.psm1:29962]: Good catch missed the rename - fixed

File: PowerValidatedSolutions.psm1 (hunk 1 above)
Comment (GaryJBlake) [reply PowerValidatedSolutions.psm1:29959]: Good catch missed the rename - fixed

**Review by tenthirtyam — APPROVED** [review]
LGTM! 🚀

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.

**Pull request merged**